pyproject.toml

```diff
@@ -21,6 +21,7 @@ html2text = "*"
 psutil = "*"
 retrying = "*"
 pyaml = "^23.9.7"
+opensearch-py = "^2.4.2"
```

Comment:
you don't need this here, just keep the one on the opensearch folder

Reply:
thanks @EmanuelCampos ! remove this.